plugins/modules/ec2_vpc_igw.py

```diff
@@ -110,6 +110,9 @@
 from ..module_utils.ec2 import ensure_ec2_tags
 from ..module_utils.ec2 import ansible_dict_to_boto3_filter_list
 from ..module_utils.tagging import boto3_tag_list_to_ansible_dict
+from time import sleep
+from time import time
+from random import randint
 
 
 @AWSRetry.jittered_backoff(retries=10, delay=10)
```

Comment:
We can probably add the retry_code here if the issue comes from `describe_igws_with_backoff` and the gw is not found. What do you suggest @tremble ?

Reply:
+1

Reply:
I guess the reason I didnt think that would help is because the failure isnt a GatewayNotFound, but rather a NoneType error, so I didnt see how catching that error code would fix the issue